README.md

```diff
@@ -100,31 +173,45 @@ Drop the content of `name-1.txt`, verify it is gone, recreate it via
 > cat name-1.txt
 ``` 
 
-The command `datalad make` does also support to just record the parameters
-that would lead to a certain computation, without actually performing the
-computation. We refer to this as *speculative computation*.
-
-To use this feature, the following configuration value has to be set:
+The `datalad make` command can also be used to perform a *speculative
+computation*. To use this feature, the following configuration value 
+has to be set:
 
 ```bash
 > git config annex.security.allow-unverified-downloads ACKTHPPT
 ```
 
-Afterward, a speculative computation can be recorded by providing the `-u` option
-(url-only) to `datalad make`.
+Afterwards, a speculative computation can be initiated by using the `-u`
+option:
```

Comment:
```suggestion
Afterwards, a speculative computation can be initiated by using the `-u / --url-only`
option:
```
This may be a personal opinion, but I like non-obvious short parameters being spelled out in both short and long form.

Reply:
This is now resolved with 5ac5ad4, although the `-u / --url-only` option will be eventually renamed.